x-pack/plugins/apm/common/utils/formatters/datetime.ts

```diff
@@ -108,7 +108,10 @@ function getFormatsAccordingToDateDifference(
 }
 
 export function asAbsoluteDateTime(
-  time: number,
+  /**
+   * timestamp in milliseconds or ISO timestamp
+   */
+  time: number | string,
```

Comment:
Turns out that we actually call this function with `string` but due to manual type we've newer noticed.